Sources/TuistLoader/Linter/ManifestLinter.swift

```diff
@@ -39,6 +53,83 @@ public class ManifestLinter: ManifestLinting {
         return issues
     }
 
+    public func lint(workspace: ProjectDescription.Workspace) -> [LintingIssue] {
+        var issues = [LintingIssue]()
+
+        for scheme in workspace.schemes {
+            if let buildAction = scheme.buildAction {
+                issues.append(contentsOf: lintWorkspaceSchemeTargets(
+                    buildAction.targets,
+                    actionType: "buildAction",
+                    scheme: scheme
+                ))
+            }
+
+            if let runAction = scheme.runAction {
+                issues.append(contentsOf: lintWorkspaceSchemeTarget(
```

Comment:
The runAction can specify an executable:

https://github.com/tuist/tuist/blob/main/Sources/ProjectDescription/RunAction.swift#L23